requirements.txt

```diff
@@ -11,20 +11,23 @@ coverage==4.5.3
 cryptography==2.6.1
 dj-database-url==0.5.0
 Django==2.1
+django-cors-headers==3.0.1
 django-cors-middleware==1.3.1
 django-extensions==2.1.0
+django-filter==2.1.0
 djangorestframework==3.9.2
 drf-yasg==1.15.0
+gunicorn==19.9.0
 idna==2.8
 inflection==0.3.1
-gunicorn==19.9.0
 isort==4.3.20
 itypes==1.1.0
 Jinja2==2.10.1
 jsonschema==3.0.1
 lazy-object-proxy==1.4.1
 MarkupSafe==1.1.1
 mccabe==0.6.1
+psycopg2==2.7.7
```

Comment:
Hi @Emmanuelaaron  l  would like to know why you add psycopg2

Reply:
Hi, @Emmanuelaaron about what @jennizalwango is asking, that second version psycopg2 that you are adding was the blocker I had when I was working on Swagger. Remember @alexxsanya raised it on my PR that it was giving him errors since there's the newer version on the next line.
If you don't remember adding it then you didn't rebase well after my PR got merged.

Reply:
Thanks @jennizalwango  and @HabibSentongo  for taking note of this. This was just an oversight. literally it's not supposed to be there.